tls/s2n_server_extensions.c

```diff
@@ -70,13 +70,8 @@ int s2n_server_extensions_send_size(struct s2n_connection *conn)
         total_size += s2n_kex_server_extension_size(conn->secure.cipher_suite->key_exchange_alg, conn);
     }
 
-    if (s2n_server_can_send_ocsp(conn)) {
-        total_size += 4;
-    }
-
-    if (s2n_server_can_send_sct_list(conn)) {
-        total_size += 4 + conn->handshake_params.our_chain_and_key->sct_list.size;
-    }
+    total_size += s2n_server_extensions_status_request_send_size(conn);
```

Comment:
In situations like this the `_size` functions are written to allow S2N_ERROR codes, but we aren't actually checking for them anyway. We should either return the `uint16_t`, or check for error codes.